spec/unit/puppet_pal_2pec.rb

```diff
@@ -755,25 +696,6 @@ def arubyfunc
         expect(result).to be(true)
       end
 
-      # Note that when run_plan moves to bolt, this needs to move there
-      it 'a plan in a module can be called with run_plan' do
-        testing_env_dir # creates the structure
-        result = Puppet::Pal.in_environment('pal_env', envpath: environments_dir, facts: node_facts) do |ctx|
-          ctx.with_script_compiler {|c| c.evaluate_string('run_plan("a::aplan")') }
-        end
-        expect(result).to eq("a::aplan value")
-      end
-
-      # Note that when run_plan moves to bolt, this needs to move there
-      it 'the envpath can have multiple entries - that are searched for the given env' do
```

Comment:
I'm going to adapt a few of these tests to run without `run_plan`, as they exercise other functionality in the PAL API.